installation/synology.md

```diff
@@ -14,8 +14,16 @@ Download SPK package from [Releases](https://github.com/openhab/openhab-syno-spk
 
 When you want to install a specific version of openHAB .spk file:
 
-1.  Login in to DiskStation Manager.2.  
+1.  Login in to DiskStation Manager.  
 2.  Go to Main Menu > Package Center.
 3.  Click on the Manual Install button.
-4.  In the window that appears, click Choose File and select the .spk file for the package you'd like to install. 
+4.  In the window that appears, click Choose File and select the openHAB .spk file for the package you'd like to install. 
 5.  On the confirmation page, if you would like the package to run immediately after installation, make sure the box next to Run after Installation is ticked. Then click Apply to start installation.
+  
+## Configuration
+
+The addons and configuration files are located at:
+```
+/volume1/public/OpenHAB2/conf
```

Comment:
why are they not in /volume1/public/openHAB2/conf ? Is there something that needs to be changed about the packaging? :-)